<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsuarioWindow</class>
 <widget class="QWidget" name="UsuarioWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Perfil de Usuario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelTitulo">
     <property name="text">
      <string>Mi Perfil</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 22px; font-weight: bold; color: #388e3c; margin: 18px 0 18px 0;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <layout class="QVBoxLayout" name="scrollLayout">
       <item>
        <widget class="QFrame" name="frameDatos">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="styleSheet">
          <string>QFrame { background: #f5fff7; border-radius: 16px; border: 1px solid #A5D6A7; padding: 22px; }</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutDatos">
          <item>
           <widget class="QLabel" name="labelNombre">
            <property name="text">
             <string>Nombre de usuario</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 15px; color: #2e7d32; font-weight: 600; margin-bottom: 2px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNombre">
            <property name="placeholderText">
             <string>Ingresa tu nombre</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 16px; padding: 8px; border-radius: 7px; border: 1px solid #bdbdbd; background: #fff; margin-bottom: 10px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCorreo">
            <property name="text">
             <string>Correo electrónico</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 15px; color: #2e7d32; font-weight: 600; margin-bottom: 2px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditCorreo">
            <property name="placeholderText">
             <string>Ingresa tu correo</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 16px; padding: 8px; border-radius: 7px; border: 1px solid #bdbdbd; background: #fff; margin-bottom: 10px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPass">
            <property name="text">
             <string>Contraseña</string>
            </property>
            <property name="styleSheet">
             <string>font-size: 15px; color: #2e7d32; font-weight: 600; margin-bottom: 2px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPass">
            <property name="placeholderText">
             <string>Dejar en blanco para no cambiar</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="styleSheet">
             <string>font-size: 16px; padding: 8px; border-radius: 7px; border: 1px solid #bdbdbd; background: #fff; margin-bottom: 10px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGuardar">
     <property name="text">
      <string>Guardar Cambios</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string>font-size: 16px; font-weight: bold; color: white; background: #388e3c; border-radius: 8px; padding: 10px 0; margin-top: 18px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
